src/graphql/main/schema.graphql

```diff
@@ -1354,7 +1355,12 @@ input UserRequestAuthCodeInput {
   phone: Phone!
 }
 
-union UserUpdate = IntraLedgerUpdate | LnUpdate | OnChainUpdate | Price
+type TransactionUpdate {
+  transaction: Transaction
```

Comment:
In general it would be helpful to return full objects that are changing from the subscriptions. If we do this, our cache is automatically updated and anything within the application that is currently looking at that object will get automatically refreshed without having to manually handled